x/twap/client/grpc/grpc_query.go

```diff
@@ -47,3 +38,12 @@ func (q Querier) ArithmeticTwapToNow(grpcCtx context.Context,
 	ctx := sdk.UnwrapSDKContext(grpcCtx)
 	return q.Q.ArithmeticTwapToNow(ctx, *req)
 }
+func (q Querier) Params(grpcCtx context.Context,
```

Comment:
proto check would fail without this change

Reply:
huh, interesting